var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -170,7 +170,7 @@ def write_library_dirs(f, dirs):
 
         # Tell numpy where to find BLAS/LAPACK libraries
         with open('site.cfg', 'w') as f:
-            if '^intel-mkl' in spec or '^intel-parallel-studio+mkl' in spec:
+            if '^intel-mkl' in spec or '^intel-parallel-studio' in spec:
```

Comment:
```suggestion
            if '^mkl' in spec:
```
This should use the `mkl` virtual dependency and be simpler.